### PR TITLE
expose tcmf num_workers to zouwu

### PR DESCRIPTION
**Pull request by yushan111**

### CODE REVIEW

File: pyzoo/test/zoo/zouwu/test_model_forecast.py [hunk 1]
```diff
@@ -263,6 +263,42 @@ def get_pred(d):
...
+                               alt_iters=2)
+        horizon = np.random.randint(1, 50)
+        # construct data
+        id = np.arange(300)
+        data = np.random.rand(300, 480)
+        input = dict({'id': id, 'y': data})
+
+        from zoo.orca import init_orca_context, stop_orca_context
+
+        init_orca_context(cores=4, spark_log_level="INFO", init_ray_on_spark=True,
```
Comment (shane-huang) [inline pyzoo/test/zoo/zouwu/test_model_forecast.py:282]: If user wants to use it, does he needs to create a spark context? Maybe we need to add a doc about how to use on a cluster (yarn) environment, i.e. where to specify yarn context, etc.

**Comment by shane-huang** [comment]
We need to tell users how to set cluster related parameters in the api docs, e.g. how to set up yarn or spark master, do we still need to call init_nn_context now? or we config using init_orca_context is enough?

File: pyzoo/test/zoo/zouwu/test_model_forecast.py (hunk 1 above)
Comment (yushan111) [reply pyzoo/test/zoo/zouwu/test_model_forecast.py:282]: init_orca_context will create a spark context. User need to specify `cluster_mode="yarn-client"` in `init_orca_context`  to run on yarn according to [doc](https://analytics-zoo.github.io/master/#Orca/context/#initialization-and-termination). UT may not be able to include that much and an example may be better.

**Comment by yushan111** [comment]
jenkins passed: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/4292/

**Pull request merged**